13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+### Metacognition is important (even though it sounds more sophisticated than it is!)
+
+Educators and educational researchers often talk about metacognition, or thinking (and ideas) about thinking, as if it is something only very sophisticated learners do, but it is truly something much more commonplace, as people (and learners) are thinking about what they are learning and doing regularly. One strategy for instructors to support metacognition is to include moments wherein learners are asked to consider what they learned and what they would like to learn more about (exit tickets can be a great way to do this, but brief period in-class can also be used). Another strategy is to help learners to recognize when it is important to ask for help: Often, when doing data science, the right question to the right person (or community) can save hours of work.
+
+### Learning strategies matter
+
+While teachers are responsible for designing learning opportunities, learners also play an important role - in their own learning! Learning strategies, according to the authors of *HPL2*, matter, including those that help students to retrieve information and summarize and explain what they have learned (for themseleves and othres). There are many specific strategies documented in [chapter four](https://www.nap.edu/read/24783/chapter/6#72) of *HPL2*. What is important for teachers of data science to know is less the specific strategies, and more the commitment to teaching learners how to learn. 
+
+In addition to strategies for learners, teaching strategies, such as how content is spaced and sequenced, can also help learners. [*Design for How People Learn*](https://www.amazon.com/s?k=design+for+how+people+learn&crid=UPT9UX7HKBS1&sprefix=design+for+how+%2Caps%2C164&ref=nb_sb_ss_i_1_15) presents these strategies, based largely about instructional design research, that may be helpful to those teaching data science.
+
+### Educators can help students to learn
+
+Educators know that how motivated learners are matters. According to the authors of *HPL2*, teachers can make an impact on how motivated learners are. Some specific things that educators can do to support learners include helping learners to set (and to work toward) goals, selecting content that is valuable and interesting to learners, helping learners to have choices and showing them how they are  in-control of their learning value, and supporting learners to feel good at and supported in what they are doing. In short, motivation matters, and may be especially important when teaching learners who do not see the value of data science (initially!).
+
+## Summary
```

Comment:
overall I think this is a fantastic start to this topic. my big wondering is how we can make it more actionable, as it reads largely as theoretical. I'm wondering if, based on how we're setting up the text, it would be more effective to provide context (as in the various paragraphs) and then 2 - 5 action steps, so that readers can choose what works for them and immediately start to put it into practice.

Reply:
great call. added as comments and notes.